app/views/layouts/application.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render "layouts/google_tag_manager" %>
 
-<%= render 'govuk_publishing_components/components/layout_for_admin',
+<%= render "govuk_publishing_components/components/layout_for_admin",
```

Comment:
Is this a requirement for the upgrade?

Reply:
Not at all! It is introduced in its very own commit, though, if that makes it any more palatable?

Reply:
Yeah I think it's fine - it might just be good to mention that this is just to follow convention (seeing as Rubocop presumably hasn't been shouting at us)?

Reply:
As an aside, Whitehall now uses `erb_lint` to lint erb files.  Maybe we should do the same in all our apps too?